e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -977,7 +990,24 @@ describeEE("scenarios > embedding > dashboard appearance", () => {
       );
     });
 
-    getIframeBody().findByText("Rows 1-21 of first 2000").should("exist");
+    getIframeBody().within(() => {
+      cy.findByText(questionDetails.name).should("exist");
+      cy.findByText("April 2022").should("exist");
+
+      // (metabase#49537)
+      chartPathWithFillColor("#509EE3").last().realHover();
```

Comment:
I re-enable these assertions following [Sasha's suggestion](https://metaboat.slack.com/archives/C064QMXEV9N/p1731615004983839?thread_ts=1731580671.819259&cid=C064QMXEV9N). I've tried a couple of other variations to fail the previous version of the code, but it still couldn't.

One thing I found on master that is different from the actual embed is that the tooltip will be located at the actual location on master, but not on Cypress. Because of this `echartsTooltip().should("be.visible")` is true which is correct.

#### Cypress
![Screenshot 2024-11-15 at 2 02 54 PM](https://github.com/user-attachments/assets/6804df2d-1bbc-47d0-81f7-1da0159a5776)


#### Actual embed
![Screenshot 2024-11-15 at 2 04 23 PM](https://github.com/user-attachments/assets/cc9cdcb9-d9e4-4d69-bc28-2ec18715df97)